src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -44,6 +42,16 @@ public function __construct(
         $this->anonymousDirectory = $anonymousDirectory ?? 'components';
     }
 
+    public static function getDefaultName(): ?string
```

Comment:
What do you think about adding an alias debug:live-component so people don't get confuse by typing debug:twig-component when they want to debug a live component ?

Reply:
Great idea!